docs/policy/memtier.md

```diff
@@ -1,70 +1,314 @@
+received via the [`node agent`](../node-agent.md), or in a fallback or forced
+[configuration file](../setup.md#using-a-local-configuration-from-a-file). These
+configuration options are
+
+  - `PinCPU`
+    * whether to pin workloads to assigned pool CPU sets
+  - `PinMemory`
+    * whether to pin workloads to assigned pool memory zones
+  - `PreferIsolatedCPUs`
+    * whether isolated CPUs are preferred by default for workloads that are
+      eligible for exclusive CPU allocation
+  - `PreferSharedCPUs`
+    * whether shared allocation is preferred by default for workloads that
+      would be otherwise eligible for exclusive CPU allocation
+
+## Policy CPU Allocation Preferences
+
+There are a number of workload properties the memtier policy actively checks to
+decide if the workload could potentially benefit from extra resource allocation
+optimizations. Unless configured differently, containers fulfilling certain
+corresponding criteria are considered eligible for these optimizations. This
+will be reflected in the assigned resources whenever that is possible at the
+time the container's creation / resource allocation request hits the policy.
+
+The set of these extra optimizations consist of
+
+  - assignment of `kube-reserved` CPUs
+  - assignment of exclusively allocated CPU cores
+  - usage of kernel-isolated CPU cores (for exclusive allocation)
+
+The policy uses a combination of the QoS class and the resource requirements of
+the container to decide if any of these extra allocation preferences should be
+applied. Containers are divided into five groups, with each group having a
+slightly different set of criteria for eligibility.
+
+  - `kube-system` group
+    * all containers in the `kube-system` namespace
+  - `low-priority` group
+    * containers in the `BestEffort or Burstable` QoS class
+  - `sub-core` group
+    * Guaranteed QoS class containers with `CPU request < 1 CPU`
+  - `mixed` group
+    * Guaranteed QoS class containers with `1 <= CPU request < 2`
+  - `multi-core` group
+    * Guaranteed QoS class containers with `CPU request >= 2`
+
+The eligibility rules for extra optimization are slightly different among these
+groups.
+
+  - `kube-system`
+    * not eligible for extra optimizations
+    * eligible to run on `kube-reserved` CPU cores
+    * always run on shared CPU cores
+  - `low-priority`
+    * not eligible for extra optimizations
+    * always run on shared CPU cores
+  - `sub-core`
+    * not eligible for extra optimizations
+    * always run on shared CPU cores
+  - `mixed`
+    * by default eligible for exclusive and isolated allocation
+    * not eligible for either if `PreferSharedCPUs` is set to true
+    * not eligible for either if annotated to opt out from exclusive allocation
+    * not eligible for isolated allocation if annotated to opt out
+  - `multi-core`
+    * CPU request fractional (`(CPU request % 1000 milli-CPU) != CPU request`):
```

Comment:
`(CPU request % 1000 milli-CPU) != CPU request`
->
`(CPU request % 1000 milli-CPU) != 0`
or
`int(CPU request / 1000 milli-CPU) != CPU request`
or
(`CPU request` is not divisible by 1000 milli-CPU)

Reply:
Fixed.